app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people.update({
+            name: self
+        })
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for person in people:
+        new_person = Person(name=person["name"], age=person["age"])
+        person_list.append(new_person)
+
+    for person in people:
+        person_instance = Person.people[person["name"]]
+        if "wife" in person and person["wife"]:
+            person_instance.wife = Person.people[person["wife"]]
+        if "husband" in person and person["husband"]:
+            person_instance.husband = Person.people[person["husband"]]
```

Comment:
Similar to the previous comment, accessing `Person.people[person['wife']]` or `Person.people[person['husband']]` assumes these keys exist in the dictionary. Add error handling to avoid `KeyError` if the spouse is not found.